src/components/DoctorsPage/DoctorsCard/Schedule/ScheduleReducer.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  ActionType,
+  IScheduleInitialState,
+  ScheduleActionTypes,
+  SetChoiceWorkTime,
+  SetDisabledItem,
+} from './types';
+
+export const initialState = {
+  choiceWorkTime: {
+    date: '',
+    time: '',
+    _id: '',
+  },
+  disabledItem: null,
+  disabledButton: false,
+  zeroing: false,
+  viewSuccessMessage: false,
+};
+
+export const setDisabledItem = (payload: null | number): SetDisabledItem => {
+  return {
```

Comment:
arrow function, you can skip return statement here and in the following functions